packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -119,12 +119,17 @@ const metadata: Meta<WaferMapArgs> = {
                 color: red;
                 font: var(${bodyFont.cssCustomProperty});
             }
+            #wafer-map {
+                width: 100%;
+                height: 100%;
+                aspect-ratio: 16 / 9;
+            }
```

Comment:
Instead of this I'd recommend letting the wafermap render at it's default size but let the user manually resize it if they want. ie:

```suggestion
            #wafer-map {
                resize: both;
                overflow: hidden;
            }
```

![resizewafer](https://user-images.githubusercontent.com/1588923/217975051-5d718419-882d-426d-bf6e-3ce8ff3b6fc1.gif)